<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>

  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>

  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>

  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>

  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>

  <property name="windowTitle">
   <string>Form</string>
  </property>

  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>

<!-- Tekst latitude en longitude -->
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Latitude</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Longitude</string>
      </property>
     </widget>
    </item>

<!-- Geef latitude en longitude in -->
    <item row="1" column="1">
     <widget class="QLineEdit" name="latitude_P">
      <property name="placeholderText">
       <string>Type latitude</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="longitude_P">
      <property name="placeholderText">
       <string>Type longitude</string>
      </property>
     </widget>
    </item>

<!-- Radius en bijbehoren -->
    <item row="2" column="0">
     <widget class="QLabel" name="label_radius">
      <property name="text">
       <string>Radius (m)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="slider_radius">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="radius">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>

<!-- Bereken aantal amenities in straal -->
    <item row="5" column="0">
     <widget class="QPushButton" name="straal_calc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>

<!-- Gebiedsanalyse -->
    <item row="6" column="0">
     <widget class="QPushButton" name="btn_gebied">
      <property name="text">
       <string>Gebiedsanalyse</string>
      </property>
     </widget>
    </item>

<!-- Routeanalyse en toebehoren -->
    <item row="7" column="0">
     <widget class="QLabel" name="label_A">
      <property name="text">
       <string>Coördinaten A</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLineEdit" name="longitude_A">
      <property name="placeholderText">
       <string>Type longitude A</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="latitude_A">
      <property name="placeholderText">
       <string>Type latitude A</string>
      </property>
     </widget>
    </item>

    <item row="7" column="1">
     <widget class="QLabel" name="label_B">
      <property name="text">
       <string>Coördinaten B</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="longitude_B">
      <property name="placeholderText">
       <string>Type longitude B</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="latitude_B">
      <property name="placeholderText">
       <string>Type latitude B</string>
      </property>
     </widget>
    </item>

    <item row="10" column="0">
     <widget class="QPushButton" name="btn_route">
      <property name="text">
       <string>Routeanalyse</string>
      </property>
     </widget>
    </item>

   </layout>
  </widget>

  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>691</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mapbox"/>
  </widget>

  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>

   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>

   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>

   <widget class="QLabel" name="tekst_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>330</width>
      <height>40</height>
     </rect>
    </property>

    <property name="autoFillBackground">
     <bool>false</bool>
    </property>

    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>

    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>

  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>450</y>
     <width>350</width>
     <height>250</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="chartWid"/>
  </widget>

 </widget>
 <resources/>
 <connections/>
</ui>